srcpkgs/felix/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'felix'
+pkgname=felix
+version=2.14.0
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="libzstd-devel libgit2-1.8-devel"
```

Comment:
```suggestion
makedepends="bzip2-devel libzstd-devel libgit2-1.8-devel"
```

Reply:
Thanks, updated now.
Just out of curiosity, how did you come up with this?

Reply:
My heuristic is to open `Cargo.lock` and search for `pkg-config`. Minus things that are nested vendor deps like `zlib` for `libgit2`.